extras/rya.export/README.md

```diff
@@ -0,0 +1,250 @@
+<!-- Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. -->
+
+# Rya Export
+
+___
+
+## Purpose
+
+Rya Export is a java application that can be used to export from a parent Rya datastore to a child Rya datastore. The changes to the child can then be synced back to the parent when the process is run again after the child datastore has collected/modified its data set. The datastores currently supported are Accumulo and MongoDB.
+
+![Figure 1: Child to Parent Merge](resources/images/child_to_parent_merge.png)
+
+## Overview
+
+At deployment time, a Rya child instance will be created and sent out where its users will create, modify, and delete data. This data from the child will then need to be synced with its parent after deployment. By default, the data only needs to be handled at the datastore level and so will not need to know the logical consistency of the higher level models it's storing (i.e. Rya logic). Some decisions that were made about the tool:
+
+  * When copying, the parent is copying its data to its child and can optionally filter its data to be after a specified timestamp.
+
+  * When merging, the data is going only ONE way: the child is re-integrating its new data with the parent Rya instance (the parent does NOT have to merge its new data into the child).
+
+  * For Accumulo datastores, since an update to a row value is just a delete followed by an add, row value updates aren't needed.
+
+  * For Accumulo datastores, special care must be taken when there's a difference between the parent's and child's column visibility in the same row. In this case, the two visibility values should be OR'ed together to produce the new visibility value to use.
```

Comment:
same here, they really don't have the option.  The visibilities will be ANDED together, not ORed